.ruby-version

```diff
@@ -1 +1 @@
-2.7.2
+2.7.5
```

Comment:
I would keep it on 2.7.2 and let the developers move it later to 2.7.5

Reply:
Weird your PR is not picking up the change that it was moved to 2.7.5 by the developers 7 days ago. Can you rebase on main, thanks?